.ci/Jenkinsfile

```diff
@@ -105,6 +105,22 @@ pipeline {
       }
     }
 
+    stage('E2e Test') {
+      steps {
+        withGithubNotify(context: 'E2e Test') {
+          cleanup()
+          withNodeEnv(){
+            dir("${BASE_DIR}"){
+              sh(label: 'install dependencies',script: 'npm i')
```

Comment:
What do you think about using `npm ci` here? It will never write to package.json or any of the package-locks: installs are essentially frozen.